src/internal/qinternal.h

```diff
@@ -39,17 +39,17 @@
 
 #define DYNAMIC_VSPRINTF(s, f) do {                                     \
         size_t _strsize;                                                \
-        for(_strsize = 1024; ; _strsize *= 2) {                         \
+        for (_strsize = 1024; ; _strsize *= 2) {                         \
             s = (char*)malloc(_strsize);                                \
-            if(s == NULL) {                                             \
+            if (s == NULL) {                                             \
                 DEBUG("DYNAMIC_VSPRINTF(): can't allocate memory.");    \
                 break;                                                  \
             }                                                           \
             va_list _arglist;                                           \
             va_start(_arglist, f);                                      \
             int _n = vsnprintf(s, _strsize, f, _arglist);               \
             va_end(_arglist);                                           \
-            if(_n >= 0 && _n < _strsize) break;                         \
+            if (_n >= 0 && _n < _strsize) break;                         \
```

Comment:
the tailing `\` are off lining up with others here too. This is happening for many of the updates. Please check them again. Maybe easier to see it on the diff UI https://github.com/wolkykim/qlibc/pull/103/files